1.preprocessing-data/ICutils.py

```diff
@@ -0,0 +1,126 @@
+import sys
+import numpy as np
+import pathlib
+from matplotlib import pyplot as plt
+from pathlib import Path
+import os
+import skimage
+
+# explicit import to PyBaSiC due to not having package support
+sys.path.append("./PyBaSiC/")
+import pybasic
+
+def load_pybasic_data(
+    channel_path: pathlib.Path,
+    channel: str,
+    file_extension: str = ".tif",
+    verbosity: bool = True,
+):
+    """
+    Load all images from a specified directory in preparation for pybasic illum correction
+
+    Parameters:
+    channels_path (pathlib.Path): path to directory where the all images for each channel are stored
+    channel (str): The name of the channel (currently one of ["GFP", "RFP", "DAPI"])
+    file_extension (str): The filename extension of the types of files to search for (default: ".tif")
+    verbosity (bool): Prints out information regarding the function running through the images (default: "True")
+
+    Returns:
+        channel_images: list of ndarrays of the loaded in images
+        image_files: list of strings of the paths to each image
+    """
+    image_files = (
+        []
+    )  # List that holds all of the paths to the images in the directory for a specific channel
```

Comment:
I think `black` may have messed this up a bit

```suggestion
    # List that holds all of the paths to the images in the directory for a specific channel
    image_files = ([])  
```

Reply:
Yup that was black's doing. I am not a fan of some of the formatting it does like that :(

Reply:
All of those are fixed now